src/components/ContentItems.astro

```diff
@@ -0,0 +1,80 @@
+---
+import Link from "@/components/Link.astro";
+import Tags, { type Tag } from "@/components/Tags.astro";
+
+export interface ContentItem {
+  title: string;
+  description?: string;
+  date?: string;
+  tags?: Tag[];
+  image?: string;
+
+  link?: string;
+  linkText?: string;
+}
+
+interface Props {
+  items?: ContentItem[];
+  id?: string;
+}
+
+const { items = [] } = Astro.props;
+---
+
+<>
+  {items.length === 0 && <p>No items found</p>}
+  {
+    items.length > 0 && (
+      <ul class="usa-card-group measure-6">
+        {items.map((n) => (
+          <li class="usa-card usa-card--flag flex-1">
+            <div class="usa-card__container">
+              <div class="usa-card__header">
+                <h2 class="usa-card__heading">
+                  <Link href={n.link}>{n.title}</Link>
+                </h2>
+                {n.date && <p class="usa-card__meta">{n.date}</p>}
+              </div>
+              {n.image && (
+                <div class="usa-card__media">
+                  <div class="usa-card__img">
+                    <img src={n.image} alt={n.title} />
```

Comment:
It's not a big deal, but I think it's more than possible that someone will want to associate alt text with the image itself, rather than imply the title is appropriate

Reply:
We have alt tag on the related image in the editor. I'll update to use it and default to title.